2021-08-04 23:12:47,896 INFO server.py Запущен сервер, порт для подключений: 7777, адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2021-08-04 23:12:52,400 INFO server.py Установлено соедение с ПК ('127.0.0.1', 58366)
2021-08-04 23:12:52,400 DEBUG server.py Получено сообщение {'action': 'presence', 'time': 1628100772.4003556, 'user': {'account_name': 'Guest'}}
2021-08-04 23:12:52,400 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1628100772.4003556, 'user': {'account_name': 'Guest'}}
2021-08-04 23:12:52,400 INFO server.py Cформирован ответ клиенту {'response': 200}
2021-08-04 23:12:52,400 DEBUG server.py Соединение с клиентом ('127.0.0.1', 58366) закрывается.
2021-08-05 23:05:47,603 INFO server.py Запущен сервер, порт для подключений: 7777, адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2021-08-05 23:05:53,795 INFO server.py Установлено соедение с ПК ('127.0.0.1', 55876)
2021-08-05 23:05:53,795 DEBUG server.py Получено сообщение {'action': 'presence', 'time': 1628186753.7956207, 'user': {'account_name': 'Guest'}}
2021-08-05 23:05:53,795 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1628186753.7956207, 'user': {'account_name': 'Guest'}}
2021-08-05 23:05:53,795 INFO server.py Cформирован ответ клиенту {'response': 200}
2021-08-05 23:05:53,795 DEBUG server.py Соединение с клиентом ('127.0.0.1', 55876) закрывается.
2021-08-05 23:10:57,166 CRITICAL config_server_log.py Критическая ошибка
2021-08-05 23:10:57,169 ERROR config_server_log.py Ошибка
2021-08-05 23:10:57,169 DEBUG config_server_log.py Отладочная информация
2021-08-05 23:10:57,169 INFO config_server_log.py Информационное сообщение
